trie.go

```diff
@@ -176,6 +184,31 @@ func (p *prefixTrie) containingNetworks(number rnet.NetworkNumber) ([]RangerEntr
 	return results, nil
 }
 
+func (p *prefixTrie) coveredNetworks(network rnet.Network) ([]RangerEntry, error) {
+	var results []RangerEntry
+	if p.hasEntry() && network.Contains(p.network.Number) {
```

Comment:
Same failure reason as above, would also show up with the added test case.  Should test by using the added correct ContainsNetwork function.

Also keep in mind that this is a trie, so if network represented at the current node is covered by the given network, all its children networks will also be covered by the given network, so any more inclusion testing is unnecessary.